tox.ini

```diff
@@ -756,24 +756,20 @@ allowlist_externals =
 commands =
     k3d version
 
-    ; bash -c "docker rm $(docker ps -aq) --force || true"
     bash -c "k3d cluster delete build || true && \
         docker volume rm k3d-build-images --force || true"
 
     bash -c 'k3d cluster create build || true && \
         k3d cluster start build'
 
-    bash -c 'rm -rf packages/grid/helm/syft/templates/ && mkdir -p packages/grid/helm/syft/templates/'
-
-    bash -c 'rm -rf packages/grid/out.txt'
-
     bash -c 'cd packages/grid && \
         [[ -n "$CONTAINER_REGISTRY" ]] && REGISTRY_FLAG="--var CONTAINER_REGISTRY=$CONTAINER_REGISTRY" || REGISTRY_FLAG="" && \
         [[ -n "$VERSION" ]] && VERSION_FLAG="--var VERSION=$VERSION" || VERSION_FLAG="" && \
-        devspace deploy --render --skip-build --build-sequential --no-warn --silent ${REGISTRY_FLAG} ${VERSION_FLAG} --kube-context "k3d-build" > out.txt'
+        devspace deploy --render --skip-build ${REGISTRY_FLAG} ${VERSION_FLAG} --kube-context "k3d-build" --no-warn --no-colors > out.txt ; \
+        printf '-%.0s' {1..100} ; echo "\nDEBUG out.txt:\n" ; cat out.txt ; printf '-%.0s' {1..100} ; echo'
```

Comment:
This will ignore the exit code of `devspace deploy`. Fix this by capturing the error code and exit accordingly